common/timing.cc

```diff
@@ -636,6 +637,147 @@ struct Timing
             }
 #endif
         }
+
+        if (ctx->debug) {
```

Comment:
Would it make sense to have a different runtime/commandline flag for this? I would imagine someone would want to run `--debug` on large designs without generating the timing graphviz, especially when you say it's CPU intensive.

Reply:
Sorry for being unclear: writing out timing.dot is not particularly CPU intensive, but running dot to render it is.

Reply:
Personally I don't like the idea of a PnR tool deciding to create files in the current directory even in debug mode, it's the kind of thing that annoys me when using vendor tools. I think this as  separate option would be better.